apps/web/.env.example

```diff
@@ -99,3 +99,6 @@ CALENDSO_ENCRYPTION_KEY=
 
 # Intercom Config
 NEXT_PUBLIC_INTERCOM_APP_ID=
+
+# Zendesk Config
+ZENDESK_KEY=
```

Comment:
Should this not be `NEXT_PUBLIC_ZENDESK_KEY`

Reply:
yes